Adding Methyl Decanoate Data

bradcarr23 opened the pull request:
3 data files from "A shock tube study of methyl decanoate autoignition at elevated pressures" from the Combustion and Flame Journal

bradcarr23 committed:
Adding Methyl Decanoate Data

bryanwweber commented:
@kyleniemeyer Can you take a look at this? This is related to the idea we had for WSSCI, although this might be all or most of the data, so we might need to pick a different fuel...

kyleniemeyer commented on methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_0.5.yaml:
```diff
@@ -0,0 +1,363 @@
+        - uncertainty-type: relative
+          uncertainty: 0.02
+      pressure-rise:
+        - 3 1/ms
```
should probably be `3.0 1/ms` to reflect significant digits given by the paper

kyleniemeyer commented on methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_0.5.yaml:
```diff
@@ -0,0 +1,363 @@
+        - uncertainty-type: relative
+          uncertainty: 0.02
+      pressure-rise:
+        - 3 1/ms
```
same comment as above, `3.0 1/ms`

kyleniemeyer commented on methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_0.5.yaml:
```diff
@@ -0,0 +1,363 @@
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 16 atm
```
`16.0 atm`

kyleniemeyer commented on methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.0.yaml:
```diff
@@ -0,0 +1,578 @@
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 14 atm
```
`14.0 atm`

kyleniemeyer commented on methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.0.yaml:
```diff
@@ -0,0 +1,578 @@
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 18 atm
```
`18.0 atm`

kyleniemeyer commented on methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.0.yaml:
```diff
@@ -0,0 +1,578 @@
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 16 atm
```
`16.0 atm`

kyleniemeyer commented on methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.5.yaml:
```diff
@@ -0,0 +1,350 @@
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 16 atm
```
`16.0 atm`

kyleniemeyer commented on methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.5.yaml:
```diff
@@ -0,0 +1,350 @@
+        - uncertainty-type: relative
+          uncertainty: 0.02
+      pressure-rise:
+        - 3 1/ms
```
`3.0 1/ms`

kyleniemeyer commented on methyl-decanoate/Methyl-Decanoate Wang xO2_0.2087_phi_1.5.yaml:
```diff
@@ -0,0 +1,350 @@
+        - uncertainty-type: relative
+          uncertainty: 0.25
+      pressure:
+        - 16 atm
```
`16.0 atm`

kyleniemeyer commented:
@bradcarr23 overall looks good! I left a few comments where significant digits got dropped from the data given in the paper (i.e., `16.0` turned into `16`). This is likely not important, as we don't use this information now, but I'd rather try and retain that information in case it somehow becomes useful in the future.

kyleniemeyer commented:
Also, I wanted to check on whether the `ignition-type` was appropriate; the paper gives this on how ignition was detected:
> The ignition delay time was defined as the time interval between shock arrival and reflection at the end wall, determined using the side wall pressure and incident shock velocity measurements, and the onset of ignition at the end wall, defined by extrapolating the maximum slope in OH* emission to the baseline.

You have the `ignition-type` set as:
```
    ignition-type: &ign
        target: OH*
        type: d/dt max
```
and I'm wondering if the `type` should instead be `d/dt max extrapolated`.

bradcarr23 commented:
Professor Niemeyer,
Sorry it has taken so long to get back to you, I have been studying for finals, and getting settled back home for break. I will fix the significant digits on the next pull request. I also believe that the ignition-type should be "d/dt max extrapolated" due to the extrapolation of the OH* emission. I will change that now and submit a new request. Thank you!

bryanwweber commented:
@bradcarr23 You can make some new commits on this branch and push it up to your copy. That will automatically update the pull request (PR)

kyleniemeyer commented:
@bradcarr23 sounds good, no rush!

bradcarr23 committed:
Edited data for methyl decanoate

bradcarr23 commented:
@kyleniemeyer @bryanwweber I have added the new files to the branch and pushed it up to my copy. Now it says that 6 files were changed, rather than just the 3 new ones. Is this correct?

bryanwweber commented:
@bradcarr23 It looks like you added the second set of files into the root directory, rather than into the `methyl-decanoate` directory. I bet if you move the new files into the subdirectory and replace the existing files, then make another commit, it will work out

bradcarr23 committed:
Updated Data for Methyl-Decanoate

Significant figures, ignition-type data, and file-version have all been updated in these new versions

kyleniemeyer commented:
@bradcarr23 ok, this looks good to accept now, thanks!
